frontend/src/components/Sidebar/Sidebar.jsx

```
@@ -1,18 +1,420 @@
+          <FaComments className="text-[#b0b0b0]" />
+          <span className="text-[15px]">Chat</span>
+        </div>
+        {/* Spaces */}
+        <div className="mt-6">
+          <div className="flex items-center justify-between px-2 mb-2">
+            <span className="text-white font-semibold text-[15px]">Spaces</span>
+            <button
+              className="w-6 h-6 flex items-center justify-center rounded bg-[#3b2bbd] hover:bg-[#4f3eea]"
+              onClick={() => setShowCreateSpace(true)}
+            >
+              <FaPlus className="text-white text-xs" />
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)